docs/source/getting_started.rst

```diff
@@ -143,10 +144,12 @@ terminal, you can launch sim with:
 
 .. code-block:: bash
 
-   . ./source.bash make run-sim
+   . ./source.bash 
+   make run-sim
 
 ``source.bash`` is an alias for the two source commands you saw above, and
-``make run-sim`` is an alias for ``ros2 launch rj_robocup sim.launch.py``.
+``make run-sim`` will launch both ER-Force's Framework (the physics simulator)
+and our stack (``ros2 launch rj_robocup sim.launch.py``).
```

Comment:
Maybe add a line mentioning the need to Ctrl-C twice? Might not be obvious to beginners

Reply:
will do, thanks!